rootio/tchain.go

```diff
@@ -0,0 +1,115 @@
+package rootio
+
+type tchain struct {
+	trees []Tree
+}
+
+func (tchain) Class() string {
+	return "TChain"
+}
+
+// Chain returns a tchain that is the concatenation of all the input Trees.
+func Chain(trees ...Tree) tchain {
+	var t tchain
+	t.trees = append(t.trees, trees...)
+	return t
+}
+
+func (t tchain) Entries() int64 {
+	var v int64 = 0
+	for i := range t.trees {
+		v = v + t.trees[i].Entries()
+	}
+	return v
+
+}
+
+// TotBytes return the total number of bytes before compression
+func (t tchain) TotBytes() int64 {
+	var v int64 = 0
+	for i := range t.trees {
+		v = v + t.trees[i].TotBytes()
+	}
+	return v
+
+}
+
+//Total number of bytes after compression
+
+func (t tchain) ZipBytes() int64 {
+	var v int64 = 0
+	for i := range t.trees {
+		v = v + t.trees[i].ZipBytes()
+	}
+	return v
+
+}
+
+func (t tchain) Branches() []Branch {
+
+	return t.trees[0].Branches()
```

Comment:
`Branches()` shouldn't panic if the slice of trees is empty.
if that slice is empty, we should just return `nil`.